src/localize/languages/pt_br.json

```
@@ -127,7 +127,13 @@
       }
     },
     "image": {
-      "src": "URL/dados da imagem estática para visualização de imagem",
+      "mode": "Modo de visualização de imagem",
+      "modes": {
+        "screensaver": "Logo Frigate embutido",
+        "url": "Imagem arbitrária especificada por URL",
+        "camera": "Instantâneo da câmera do Home Assistant da entidade da câmera"
```

```suggestion
        "camera": "Instantâneo da câmera do Home Assistant, da entidade de câmera"
```

"Instantâneo" I really don't know if it's any better than simply "snapshot". Anyway, I'm just thinking out loud.